### DISCUSSION TITLE
Elastic Itself Drop

### DISCUSSION THREAD

**Discussion by dunakov** [Jeroen-G/Explorer#130] [Q&A]
Has anyone encountered auto-removal of elements from the index? On my production server, the index is cleared by itself. Sometimes removed completely, sometimes partially. And I don't know the reason, since there are no errors in the logs of the docker container.

**Comment by Jeroen-G** [comment]
Is the docker container perhaps re-created and thus lost all its data?

**Reply by dunakov** [reply]
I think  it was Machine Learning. When i cheacked  logs, I saw that deletion was performed by a service marked ML. ML = Machine Learning. 
I reconfigured elastic and now I monitor its condition.
